tokio-threadpool/src/builder.rs

```diff
@@ -99,11 +103,14 @@ impl Builder {
 
         Builder {
             pool_size: num_cpus,
+            max_blocking: 100,
```

Comment:
is there a reason this isn't stored on the `Config`?

Reply:
The `Config` object is passed into the pool. Values not in `Config` don't need to be stored and are just used to initialize state (in this case, populate the stack of sleeping threads).